<launch>
 <!-- Turtlesim Node-->
  <node pkg="turtlesim" type="turtlesim_node" name="sim"/>

 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="turtle_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
   
  </node>

 <!-- joy turtle node -->
  <param name="forward_button" value="1" type="int"/>
  <param name="angular_button" value="0" type="int"/>
  <param name="control_button" value="0" type="int"/>
  <param name="mult_speed_button" value="4" type="int"/>
  <param name="pub_topic_" value="turtle1/cmd_vel" type="string"/>
  <param name="sub_topic_" value="joy" type="string"/>
  <node pkg="joy_turtle" type="JoyTurtle" name="teleop" output="screen" launch-prefix="gnome-terminal -e"/>
</launch>

